tensorboard/components/tf_imports/BUILD

```diff
@@ -135,14 +135,25 @@ tf_web_library(
     ],
 )
 
+tf_web_library(
```

Comment:
Might be worth a comment like "separated out from :d3 target to apply `strip_prefix`"?  (Especially useful internally when `strip_prefix` gets removed.)